generators/constants.js

```diff
@@ -0,0 +1,11 @@
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import { fileURLToPath } from 'url';
+
+export const DEFAULT_DATA_ACCESS = 'activeRecord';
+export const QUARKUS_VERSION = '3.3.0';
```

Comment:
```suggestion
export const QUARKUS_VERSION = '3.5.0';
```
Or this PR might fix it: https://github.com/jhipster/generator-jhipster-quarkus/pull/299

Reply:
We should avoid mixing jhipster v8 with features like quarkus update.

Reply:
I agree.